frontend/src/components/regions/CardDetailsModal/index.js

```diff
@@ -80,6 +82,10 @@ function CardDetailsModalUnconnected({
     openReviewFormModal({ cardId: project.agileCard });
   };
 
+  const handleClickSetDueTime = () => {
+    openDueTimeFormModal({ cardId: project.agileCard });
```

Comment:
Can we not set due times for topics? Why do we need to make use of the project here?

why not just:
```
    openDueTimeFormModal({ cardId });
```

cardId is a prop. A card can be associated with a project or a topic